docs/how-to/peering-with-content-providers.md

```diff
@@ -72,6 +72,12 @@ This list is provided for informational purposes only. The IPFS Project does not
 | ------- | --------- |
 |`12D3KooWGtYkBAaqJMJEmywMxaCiNP7LCEFUAFiLEBASe232c2VH`|`/dns4/bitswap.filebase.io/tcp/443/wss`|
 
+### Functionland Fula Network
+
+| Peer ID | Addresses |
+| ------- | --------- |
+|`12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb`|`/dns4/ipfs.cloud.fx.land/tcp/443/wss`|
```

Comment:
@ehsan6sha this address does not seem to work correctly.

kubo is unable to connect to it:

```console
$ ipfs swarm connect /dns4/ipfs.cloud.fx.land/tcp/443/wss/p2p/12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb
Error: connect 12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb failure: failed to dial: failed to dial 12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb: all dials failed
  * [/ip4/15.223.126.226/tcp/443/tls/sni/ipfs.cloud.fx.land/ws] unexpected EOF
```

fyi got more info  with  [vole](https://github.com/ipfs-shipyard/vole):

```console
$ go install github.com/ipfs-shipyard/vole@latest
$ vole libp2p identify /dns4/ipfs.cloud.fx.land/tcp/443/wss/p2p/12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb
panic: failed to dial: failed to dial 12D3KooWEGy1b9QLiQ4kyjk9GhF3vcWj3aQyVzfbuW7JeS1NdWhb: all dials failed
  * [/ip4/15.223.126.226/tcp/443/tls/sni/ipfs.cloud.fx.land/ws] malformed HTTP response "\x13/multistream/1.0.0"
```